app/services/listing_index_service/data_types.rb

```diff
@@ -17,7 +17,7 @@ module ListingIndexService::DataTypes
     [:per_page, :to_integer, :mandatory, gte: 1],
     [:keywords, :string, :optional],
     [:categories, :array, :optional],
-    [:listing_shape_id, :fixnum, :optional],
+    [:listing_shape_id, :array, :optional],
```

Comment:
@rap1ds yes, that's a good point so I'll do the renaming in sphinx spesific code.